### DISCUSSION TITLE
Releasing Kyuubi v1.2.0

### DISCUSSION THREAD

**Discussion by ulysses-you** [apache/kyuubi#640] [General]
Prepare to release Kyuubi v1.2.0 in this week.

**Comment by ulysses-you** [comment]
FYI @yaooqinn @pan3793 @turboFei

**Comment by yaooqinn** [comment]
LGTM

**Comment by zhenjiaguo** [comment]
Hi, I have a small question, where is the v1.2 Release Note?
I see the [release page](https://github.com/apache/incubator-kyuubi/releases?page=1) says https://kyuubi.apache.org/releases.html has a new release. But there is no v1.2 Release Note.
Thanks for reply.

**Reply by ulysses-you** [reply]
hi @zhenjiaguo , the v1.2 version is released before we join Apache so it is legacy. Please see the latest version [v1.3.1-incubating](https://kyuubi.apache.org/release/1.3.1-incubating.html).

Hope it can help you.
